oemoflex/model/model_structure.py

```diff
@@ -140,6 +140,18 @@ def create_default_data(
         component_attrs, select_components
     )
 
+    transmission_without_link = [
```

Comment:
Very nice and readable!
And general. Does not onyl apply to components that are called electricity-transmission. But all kinds of links. I would adapt the variable name and error message.
Apart from that, very nice!

Reply:
Good point! I rephrased the error message with commit https://github.com/rl-institut/oemoflex/pull/70/commits/3337175805d98e5187f220fcea48ab7b3ff4ec62.